IdentitySdkCore/IdentitySdkCore/Classes/ReachFiveApi.swift

```diff
@@ -301,4 +301,19 @@ public class ReachFiveApi {
     func createUrl(path: String) -> String {
         return "https://\(sdkConfig.domain)\(path)"
     }
+    
+    public func authorize(options: [String:String]) -> String {
```

Comment:
1) Method naming: this just builds the URL, is this correct? 2) Can appending query params really throw?

Reply:
Yes it's correct, we must build the URL to send it to the redirect Class